src/components/Table/GridTable.test.tsx

```diff
@@ -803,6 +817,58 @@ describe("GridTable", () => {
     expect(row(r, 1)).toBeUndefined();
   });
 
+  it("collapsing a parent row only rerenders the collapse cell", async () => {
+    // Given a parent with a child and grandchild
+    const rows: GridDataRow<NestedRow>[] = [
+      {
+        ...{ kind: "parent", id: "p1", name: "parent 1" },
+        children: [
+          {
+            ...{ kind: "child", id: "p1c1", name: "child p1c1" },
+            children: [{ kind: "grandChild", id: "p1c1g1", name: "grandchild p1c1g1" }],
+          },
+        ],
+      },
+    ];
+    const renderedCells: [string, string][] = [];
+    function renderCounter(kind: string, column: string) {
+      renderedCells.push([kind, column]);
+    }
+    const r = await render(<GridTable<NestedRow> columns={nestedColumns(renderCounter)} rows={rows} />);
+    renderedCells.length = 0;
+    expect(cell(r, 0, 0)).toHaveTextContent("-");
+    // When the parent is collapsed
+    click(cell(r, 0, 0).children[0] as any);
+    // Then only the parent row is still shown
+    expect(cell(r, 0, 0)).toHaveTextContent("+");
```

Comment:
@bdow - The test currently fails here... saying it is still showing a `-`. Do you have any thoughts on how I can test that it has updated. Interestingly, the lines below do pass, indicating that the GridTable did rerender to remove rows, but seems like it did not rerender the cell (yet?)